src/test/ui/consts/const_let_eq.rs

```diff
@@ -5,6 +5,7 @@ struct Bar<T> { x: T }
 struct W(u32);
 struct A { a: u32 }
 
+#[allow(redundant_semicolons)]
```

Comment:
These `allow`s are no longer necessary because `redundant_semicolons` in in the `unused` group.

Reply:
When I removed it I got:
```
unexpected errors (from JSON output): [
    Error {
        line_num: 22,
        kind: Some(
            Warning,
        ),
        msg: "22:5: 22:8: unnecessary trailing semicolons [redundant_semicolons]",
    },
]
```

Reply:
Ah, ok, they are necessary in `run-pass` tests, yes.